modules/instrument_builder/jsx/react.questions.js

```diff
@@ -631,6 +631,15 @@ var AddElement = React.createClass({
         });
         hasError = true;
       }
+      if (minYear > 9999 || minYear < 1900 || maxYear > 9999 || maxYear < 1900) {
+        let temp = (this.state.error) ? this.state.error : {};
+
+        temp.dateOption = "The year must have exactly 4 digits.(between 1900 and 9999)";
```

Comment:
Please change the error message to "The year must have exactly 4 digits. Please choose an integer number between 1900 and 9999."